packages/commons/src/config/readmodelSQLDbConfig.ts

```diff
@@ -2,25 +2,40 @@ import { z } from "zod";
 
 export const ReadModelSQLDbConfig = z
   .object({
-    READMODEL_SQL_DB_HOST: z.string(),
-    READMODEL_SQL_DB_NAME: z.string(),
-    READMODEL_SQL_DB_USERNAME: z.string(),
-    READMODEL_SQL_DB_PASSWORD: z.string(),
-    READMODEL_SQL_DB_PORT: z.coerce.number().min(1001),
+    READMODEL_SQL_DB_HOST: z.string().default("localhost"),
+    READMODEL_SQL_DB_NAME: z.string().default("test"),
+    READMODEL_SQL_DB_USERNAME: z.string().default("test"),
+    READMODEL_SQL_DB_PASSWORD: z.string().default("test"),
+    READMODEL_SQL_DB_PORT: z.coerce.number().min(1001).default(6002),
```

Comment:
Should we set as default the values that connect to the local dev readmodel DB?
```suggestion
    READMODEL_SQL_DB_HOST: z.string().default("localhost"),
    READMODEL_SQL_DB_NAME: z.string().default("root"),
    READMODEL_SQL_DB_USERNAME: z.string().default("root"),
    READMODEL_SQL_DB_PASSWORD: z.string().default("root"),
    READMODEL_SQL_DB_PORT: z.coerce.number().min(1001).default(6002),
```

Reply:
The higher-ups say "root" cannot be hard coded

Reply:
Mmh, it's already hardcoded in all the `.env` files 😄  
It's weird that here we set different values from the ones we hardcode the env files for local dev, if the problem is the word "root" we might as well change the local user/password in the docker-compose and update all the environment files

Reply:
@micdes-pagopa Can you clarify this? I'm honestly pretty confused on the matter. I'd also like to keep it aligned with the `docker-compose` file?

Reply:
During the latest security assessment it was reported to me that the secret scanning tools were complaining about these defaults in the code (I'm also pretty surprised for the same reasons as @ecamellini 🙃 ). To avoid these warnings I asked @paolomanca-pagopa  if it was possible to change the default, but if the impact is so large across multiple layers let's keep the as-is

Reply:
I [track](https://pagopa.atlassian.net/browse/PIN-6791) later cleanup of this default values in a task. The problem of parse process.env in readmodel-models could be fixed [in other way](https://pagopa.atlassian.net/browse/PIN-6531).